app/views/govuk_publishing_components/components/_layout_header.html.erb

```diff
@@ -11,7 +11,7 @@
   end
 %>
 
-<header class="gem-c-layout-header govuk-header gem-c-layout-header--<%= environment %> govuk-grid-row" role="banner" data-module="govuk-header">
+<header class="gem-c-layout-header govuk-header gem-c-layout-header--<%= environment %>" role="banner" data-module="govuk-header">
```

Comment:
**Reviewing commit by commit**

Obviously the `govuk-grid-row` class here was causing a problem but beneath it are some column classes, are they wrapped by a grid row somewhere higher up?